package.json

```diff
@@ -5,7 +5,8 @@
     "ng": "ng",
     "start": "ng serve",
     "start-docker": "ng serve --host=0.0.0.0",
-    "build": "ng build --prod=true --progress=true",
+    "build": "ng build --prod=true --progress=true --stats-json",
```

Comment:
@danieltolmon i would prefer ti have this later on in a new branch ;)

Reply:
Okai, I create a branch to add it